test/test_core.py

```diff
@@ -1089,20 +1114,19 @@ def test_wcslen(self):
   def test_regex(self):
     self.do_core_test('test_regex.c')
 
-  @no_wasm64('MEMORY64 does not yet support SJLJ')
   @also_with_standalone_wasm(wasm2c=True, impure=True)
   def test_longjmp_standalone(self):
     self.do_core_test('test_longjmp.c')
 
-  @with_both_eh_sjlj
+  @with_both_longjmp
   def test_longjmp(self):
     self.do_core_test('test_longjmp.c')
 
   @no_wasm64('MEMORY64 does not yet support SJLJ')
   def test_longjmp_with_and_without_exceptions(self):
     # Emscripten SjLj with and without Emscripten EH support
     self.set_setting('SUPPORT_LONGJMP', 'emscripten')
-    self.set_setting('DEFAULT_TO_CXX') # See comments on @with_both_eh_sjlj
+    self.set_setting('DEFAULT_TO_CXX') # See comments on @with_both_longjmp
```

Comment:
There isn't anything on that in `with_both_longjmp` - I think this should point to `with_both_eh_sjlj`?